PAY-6243: Update Apple Pay and CC container prop docs (#518)
commerce-docs/microsite-commerce-storefront: src/content/docs/dropins/payment-services/containers/credit-card.mdx

| Option | Type | Req? | Description |
|--------|------|------|-------------|
| `getCartId` | function | Yes | Should return a promise that resolves to the shopper's cart ID. |

> | `creditCardFormRef` | object | Yes | Credit card form reference. Initially, `{ current: null }` should be passed. Once rendered, the credit card container will set the `current` property to an `{ validate: () => boolean; submit: () => Promise<void> }` object, which parent containers can use to (programmatically) validate and submit the credit card form. Any error that happens during the payment flow won't only propagate to onError but submit will also return a rejected promise.  |

Reviewer: ```suggestion
| `creditCardFormRef` | object | Yes | Credit card form reference. Initially, `{ current: null }` should be passed. Once rendered, the credit card container will set the `current` property to an `{ validate: () => boolean; submit: () => Promise<void> }` object, which parent containers can use to (programmatically) validate and submit the credit card form. Any error that happens during the payment flow will not only propagate to onError, but submit will also return a rejected promise.  |
```

> | `creditCardFormRef` | object | Yes | Credit card form reference. Initially, `{ current: null }` should be passed. Once rendered, the credit card container will set the `current` property to an `{ validate: () => boolean; submit: () => Promise<void> }` object, which parent containers can use to (programmatically) validate and submit the credit card form. Any error that happens during the payment flow will not only propagate to onError, but submit will also return a rejected promise.  |

Reviewer 2: ```suggestion
| `creditCardFormRef` | object | Yes | Credit card form reference. Initially, `{ current: null }` should be passed. Once rendered, the credit card container will set the `current` property to an `{ validate: () => boolean; submit: () => Promise<void> }` object, which parent containers can use to (programmatically) validate and submit the credit card form. Any error that happens during the payment flow will not only propagate to `onError`, but will also cause the promise returned by `submit()` to be rejected. |
```

> | `onSuccess` | function | No | Called when the payment flow finishes successfully. The function receives a single argument containing a cartId property. |

Reviewer 2: ```suggestion
| `onSuccess` | function | No | Called when the payment flow finishes successfully. The function receives an `{ cartId: string }` object as an argument. |
```
